dataactvalidator/config/sqlrules/a10_appropriations.sql

```diff
@@ -1,8 +1,14 @@
+WITH appropriation_a10 AS 
```

Comment:
B9 uses `WITH [table-name]_[rule-name]_[submission-id]` in order to ensure we avoid any conflicts while multiple rules are running simultaneously. Can you make sure that doing it this way `WITH appropriation_a10` won't cause any issues when 2 submission are running the same rule at the same time?

If it won't cause problems, then this can stay as is.